src/desktop/config.coffee

```diff
@@ -32,10 +32,11 @@ module.exports =
   DEPLOY_ENV: null
   DEFAULT_CACHE_TIME: 3600
   DISABLE_IMAGE_PROXY: false
-  ENABLE_INSTANT_PAGE: true,
+  ENABLE_INSTANT_PAGE: true
```

Comment:
By any strange chance, would this cause a behavior change?

I'm noticing that `ENABLE_INSTANT_PAGE=true` in staging but is unset in prod, by the way.

Reply:
Ha, it shouldn't. I just thought it was confusing since none of the others had them.

![](http://static.damonzucconi.com/_capture/71AgLLLXF0Ms.png)

The only time it matters apparently is if it's a trailing comma... which I find insane.

![](http://static.damonzucconi.com/_capture/nDzRkbdmJFcz.png)

Reply:
I also don't really know anything about `ENABLE_INSTANT_PAGE` — @damassi ?

Reply:
It toggles this https://instant.page -- which is enabled on older pages but [disabled on new](https://github.com/artsy/force/blob/master/src/desktop/apps/artsy-v2/server.tsx#L83-L85)